pallets/nfts/src/tests.rs

```diff
@@ -4520,6 +4484,143 @@ fn cancel_collection_approval_works() {
 	});
 }
 
+#[test]
+fn clear_collection_approvals_works() {
```

Comment:
Not related to this PR.

Reply:
I take my comment back, this should be reverted.